### ISSUE TITLE
Generate requirements around export functionality

### ISSUE THREAD

**Issue by pkalita-lbl** [microbiomedata/nmdc-field-notes#96]
We've talked about having "export functionality". This issue to define a little more precisely what we mean by that before we start implementing anything. Some questions to ask ourselves:

* Are we exporting at the study level, sample level, or both?
* What format(s) are we exporting? JSON? CSV? Multiple files ZIP'd together? What about images?
* Where do the exported file(s) go? Device download? Google drive? Email?

**Comment by eecavanna** [comment]
I'm wondering whether there an analog of this in the existing submission portal/data portal. In other words, I'm wondering whether either of those things allows users to export studies/samples as files (independently of this Field Notes app). If so, I think I would consider exporting them in the same format.

**Comment by pkalita-lbl** [comment]
Yes, Montana briefly brought this up at yesterday's call. The submission portal allows you to import/export _just_ the sample metadata

![image](https://github.com/microbiomedata/nmdc-field-notes/assets/102547565/b5fb1206-5a48-4403-8bab-b3390e625f8a)

That's all done via the portal's frontend code, so there isn't any API that could be reused by the app. The other challenge is whether/how to deal with the study metadata.

**Comment by pkalita-lbl** [comment]
See Leah's notes here: https://docs.google.com/document/d/1AB4D1vnix8mIBl_b2WlqivFQbtOS28AiowATu_z2u3w/edit?tab=t.0
